### PR TITLE
KAFKA-3257: disable bootstrap-test-env.sh --colour option

### PR DESCRIPTION
**Pull request by zhuchen1018**
@becketqin, when you get a chance, could you take a look at the patch?

### CODE REVIEW

File: tests/bootstrap-test-env.sh [hunk 1]
```diff
@@ -32,7 +32,7 @@ else
 fi
 
 echo "Checking Vagrant installation..."
-vagrant_version=`vagrant --version | egrep -o "[0-9]+\.[0-9]+\.[0-9]+"`
+vagrant_version=`vagrant --version | egrep --colour=never -o "[0-9]+\.[0-9]+\.[0-9]+"`
```
Comment (granthenke) [inline tests/bootstrap-test-env.sh]: This is just the first check, but egrep is used other places. Should the other calls be updated?

Comment (granders) [inline tests/bootstrap-test-env.sh]: To avoid needing to modify every `egrep` command manually, consider using

`export GREP_OPTIONS='--color=never'`

Comment (zhuchen1018) [inline tests/bootstrap-test-env.sh]: Thanks for the suggestion! The file has been updated.

**Comment by becketqin** [comment]
LGTM. @ewencp do you mind taking a look? 

We are also using egrep in compatibility test. @apovzner is working on that. Maybe we can fix the compatibility test in that ticket as well.

**Comment by ewencp** [comment]
LGTM.

The use in the test is a bit different because it'll happen on one of the VMs. This is still potentially a concern, but there's probably a lot less variability in the default setting for use of color in VMs than there is on the host machine for running the tests (where developers may have done a lot of things to customize their environment). I'd hold off on ensuring we handle this in all code that runs in the test VMs until we know somebody is encountering since I think it'll probably be unusual in that case.

**Comment by ewencp** [comment]
And thanks for the patch @zhuchen1018!

**Pull request closed without merging**